dir.props

```diff
@@ -10,11 +10,21 @@
     <!-- true if we have bootstrapped buildtools (usually on an unsupported platform -->
     <_IsBootstrapping Condition="'$(BootstrapBuildToolsDir)' != ''">true</_IsBootstrapping>
 
-    <PortableBuild Condition="'$(OS)' == 'Windows_NT'">true</PortableBuild>
-    <PortableBuild Condition="'$(PortableBuild)' == ''">false</PortableBuild>
-    <UseSystemLibraries Condition="'$(UseSystemLibraries)' == '' AND '$(PortableBuild)' != 'true'">true</UseSystemLibraries>
-    <UseSystemLibunwind Condition="'$(UseSystemLibunwind)' == ''">$(UseSystemLibraries)</UseSystemLibunwind>
     <UseStableVersions Condition="'$(UseStableVersions)' == ''">true</UseStableVersions>
+
+    <!-- new supported portable/nonportable options, default based on old options if set, otherwise just regular defaults -->
+    <BuildPortableRuntime Condition="'(BuildPortableRuntime)' == '' and '$(PortableBuild)' == 'false'">false</BuildPortableRuntime>
+    <BuildPortableRuntime Condition="'(BuildPortableRuntime)' == '' and '$(PortableBuild)' == 'true'">true</BuildPortableRuntime>
+    <BuildPortableRuntime Condition="'(BuildPortableRuntime)' == ''">false</BuildPortableRuntime>
+    <BuildNonPortableRuntime Condition="'(BuildNonPortableRuntime)' == '' and '$(PortableBuild)' == 'false'">true</BuildNonPortableRuntime>
+    <BuildNonPortableRuntime Condition="'(BuildNonPortableRuntime)' == '' and '$(PortableBuild)' == 'true'">false</BuildNonPortableRuntime>
+    <BuildNonPortableRuntime Condition="'(BuildNonPortableRuntime)' == ''">true</BuildNonPortableRuntime>
+    <BuildPortableSdk Condition="'(BuildPortableSdk)' == '' and '$(PortableBuild)' == 'false'">false</BuildPortableSdk>
+    <BuildPortableSdk Condition="'(BuildPortableSdk)' == '' and '$(PortableBuild)' == 'true'">true</BuildPortableSdk>
+    <BuildPortableSdk Condition="'(BuildPortableSdk)' == ''">false</BuildPortableSdk>
```

Comment:
Is BuildPortableSdk  & BuildNonPortableSdk used anywhere?

Reply:
Yeah, the idea is that these are the new control variables for both portable and non-portable builds: https://github.com/dotnet/source-build/pull/1530/files#diff-92e85f9e6b0e59ac672e4c0aeb30cbe5R50.  Then you can either turn these on and off manually, or our legacy variables will default them as before.